ibm/mas_devops/roles/suite_db2_setup_for_manage/templates/setupdb.sh.j2

```diff
@@ -25,17 +25,37 @@ db2 update db cfg for {{ db2_dbname }} using dft_table_org ROW
 db2 update db cfg for {{ db2_dbname }} using AUTO_MAINT ON DEFERRED
 db2 update db cfg for {{ db2_dbname }} using AUTO_TBL_MAINT ON DEFERRED
 db2 update db cfg for {{ db2_dbname }} using AUTO_RUNSTATS ON DEFERRED
-db2 update db cfg for {{ db2_dbname }} using AUTO_REORG ON DEFERRED
-db2 update db cfg for {{ db2_dbname }} using AUTO_DB_BACKUP ON DEFERRED
+
+### db2 update db cfg for {{ db2_dbname }} using AUTO_REORG ON DEFERRED
+db2 update db cfg for {{ db2_dbname }} using AUTO_REORG OFF DEFERRED
+
+
+### db2 update db cfg for {{ db2_dbname }} using AUTO_DB_BACKUP ON DEFERRED
+db2 update db cfg for {{ db2_dbname }} using AUTO_DB_BACKUP OFF DEFERRED
+
 db2 update db cfg for {{ db2_dbname }} using CATALOGCACHE_SZ 800 DEFERRED
 db2 update db cfg for {{ db2_dbname }} using CHNGPGS_THRESH 40 DEFERRED
 db2 update db cfg for {{ db2_dbname }} using DBHEAP AUTOMATIC
 db2 update db cfg for {{ db2_dbname }} using LOCKLIST AUTOMATIC DEFERRED
 db2 update db cfg for {{ db2_dbname }} using LOGBUFSZ 1024 DEFERRED
 db2 update db cfg for {{ db2_dbname }} using LOCKTIMEOUT 300 DEFERRED
-db2 update db cfg for {{ db2_dbname }} using LOGPRIMARY 20 DEFERRED
-db2 update db cfg for {{ db2_dbname }} using LOGSECOND 100 DEFERRED
-db2 update db cfg for {{ db2_dbname }} using LOGFILSIZ 8192 DEFERRED
+
+### db2 update db cfg for {{ db2_dbname }} using LOGPRIMARY 20 DEFERRED
+db2 update db cfg for {{ db2_dbname }} using LOGPRIMARY 100 DEFERRED
+
+### db2 update db cfg for {{ db2_dbname }} using LOGSECOND 100 DEFERRED
+db2 update db cfg for {{ db2_dbname }} using LOGSECOND 156 DEFERRED
+
+
+### db2 update db cfg for {{ db2_dbname }} using LOGFILSIZ 8192 DEFERRED
+db2 update db cfg for {{ db2_dbname }} using LOGFILSIZ 50000 DEFERRED
+
+db2 update db cfg for {{ db2_dbname }} using MIRRORLOGPATH /mnt/backup/MIRRORLOGPATH   ### Added
```

Comment:
No need to add `### Added` comments to new lines, remove the comment please.  We can see new lines in the diff tools.

Reply:
Removed comment `### Added`